lib/utils/ember.js

```diff
@@ -89,11 +89,28 @@ function isDSModel(node) {
 }
 
 function isModuleByFilePath(filePath, module) {
-  const fileName = `${module}.js`;
+  if (filePath.indexOf('app/') === -1) { return false; }
+
+  const fileName = `/${module}.js`;
   const folderName = `${module}s`;
```

Comment:
Yeah, you're probably right. I had a feeling that it might be a little bit cheesy too, but wanted to get conceptual feedback as well before further refactoring. I'll take your idea into consideration and I'll try to address it :)